peps/pep-0719.rst

```diff
@@ -42,11 +42,10 @@ Actual:
 - 3.13.0 alpha 4: Thursday, 2024-02-15
 - 3.13.0 alpha 5: Tuesday, 2024-03-12
 - 3.13.0 alpha 6: Tuesday, 2024-04-09
+- 3.13.0 beta 1: Wednesday, 2024-05-08
```

Comment:
Oh, and let's keep the feature freeze marker (comparing 3.12: https://peps.python.org/pep-0693/)

```suggestion
- 3.13.0 beta 1: Wednesday, 2024-05-08 (No new features beyond this point.)
```